astropy/cosmology/io/latex.py

```diff
@@ -0,0 +1,72 @@
+import astropy.units as u
+from astropy.cosmology.connect import readwrite_registry
+from astropy.cosmology.core import Cosmology
+from astropy.cosmology.parameter import Parameter
+from astropy.table import QTable
+
+from .table import to_table
+
+_FORMAT_TABLE = {
+    "H0": "$$H_0 \r{[\\mathrm{km\\,s^{-1}\\,Mpc^{-1}}]}$$",
+    "Om0": "$$\\Omega_{m,0}$$",
+    "Ode0": "$$\\Omega_{\\Lambda,0}$$",
+    "Tcmb0": "$$T_{0} \r{[\\mathrm{K}]}$$",
+    "Neff": "$$N_{eff}$$",
+    "m_nu": "$$m_{nu} \r{[\\mathrm{eV}]}$$",
+    "Ob0": "$$\\Omega_{b,0}$$",
+    "w0": "$$w_{0}$$",
+    "wa": "$$w_{a}$$",
+    "wz": "$$w_{z}$$",
+    "wp": "$$w_{p}$$",
+    "zp": "$$z_{p}$$",
+}
+
+
+def write_latex(
+    cosmology, file, *, overwrite=False, cls=QTable, latex_names=True, **kwargs
+):
+    r"""Serialize the |Cosmology| into a LaTeX.
+
+    Parameters
+    ----------
+    cosmology : `~astropy.cosmology.Cosmology` subclass instance
+    file : path-like or file-like
+        Location to save the serialized cosmology.
+
+    overwrite : bool
+        Whether to overwrite the file, if it exists.
+    cls : type (optional, keyword-only)
+        Astropy :class:`~astropy.table.Table` (sub)class to use when writing.
+        Default is :class:`~astropy.table.QTable`.
+    **kwargs
+        Passed to ``cls.write``
+
+    Raises
+    ------
+    TypeError
+        If kwarg (optional) 'cls' is not a subclass of `astropy.table.Table`
+    """
+    # Set cosmology_in_meta as false for now since there is no metadata being kept
+    table = to_table(cosmology, cls=cls, cosmology_in_meta=False)
+
+    cosmo_cls = type(cosmology)
+    for name, col in table.columns.copy().items():
+        param = getattr(cosmo_cls, name, None)
+        if not isinstance(param, Parameter) or param.unit in (None, u.one):
+            continue
+        # Replace column with unitless version
+        table.replace_column(name, (col << param.unit).value, copy=False)
+
+        # Convert parameter names to LaTeX format
+        if name in ("cosmology", "name"):
+            pass
+        else:
+            table.rename_column(name, _FORMAT_TABLE.get(name))
```

Comment:
For safety, 
```suggestion
            table.rename_column(name, _FORMAT_TABLE.get(name, name))
```
will default to the original name if it's not in the format table.